app/assets/stylesheets/users.scss

```diff
@@ -3,6 +3,9 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 $red: #ff0000;
 
+body {
+  background-color: #fdf8f5;
```

Comment:
Color literals like `#fdf8f5` should only be used in variable declarations; they should be referred to via variable everywhere else.